src/test/java/de/zalando/aruha/nakadi/security/OffModeAuthenticationTest.java

```diff
@@ -1,46 +1,31 @@
 package de.zalando.aruha.nakadi.security;
 
 
-import com.google.common.collect.ImmutableList;
 import de.zalando.aruha.nakadi.config.SecuritySettings;
+import java.util.stream.Stream;
 import org.junit.Test;
 
-import java.util.List;
-
-import static org.springframework.http.HttpMethod.DELETE;
-import static org.springframework.http.HttpMethod.GET;
-import static org.springframework.http.HttpMethod.POST;
-import static org.springframework.http.HttpMethod.PUT;
-
 public class OffModeAuthenticationTest extends AuthenticationTest {
 
     static {
         authMode = SecuritySettings.AuthMode.OFF;
     }
 
-    private static final List<Endpoint> endpoints = ImmutableList.of(
-            new Endpoint(GET, "/event-types"),
-            new Endpoint(POST, "/event-types"),
-            new Endpoint(GET, "/event-types/foo"),
-            new Endpoint(PUT, "/event-types/foo"),
-            new Endpoint(DELETE, "/event-types/foo"),
-            new Endpoint(POST, "/event-types/foo/events"),
-            new Endpoint(GET, "/event-types/foo/events"),
-            new Endpoint(GET, "/event-types/foo/partitions"),
-            new Endpoint(GET, "/event-types/foo/partitions/bar"),
-            new Endpoint(GET, "/health"),
-            new Endpoint(GET, "/metrics"));
-
     @Test
     public void offAuthMode() {
-        endpoints.stream().forEach(endpoint -> {
-            try {
-                mockMvc.perform(endpoint.toRequestBuilder()).andExpect(STATUS_NOT_401_OR_403);
-            }
-            catch (AssertionError | Exception e) {
-                throw new AssertionError("Exception/AssertionError when calling endpoint: " + endpoint, e);
-            }
-        });
+        Stream.concat(endpoints.stream(), endpointsForUidScope.stream())
+                .forEach(this::checkHasCorrectResponseStatus);
+    }
+
+    public void checkHasCorrectResponseStatus(Endpoint endpoint) {
+        try {
+            mockMvc.perform(endpoint.toRequestBuilder()).andExpect(STATUS_NOT_401_OR_403);
+        } catch (Exception e) {
+            throw new AssertionError("Exception when calling endpoint: " + endpoint, e);
+        } catch (AssertionError e) {
+            throw e;
```

Comment:
If you just throw exception here - then this block doesn't really make sense, because you just throw what you caught. What can add value here is to create and throw AssertionError that contains the endpoint (like few lines above), because if mockMvc check fails - it does not provide the information about the endpoint.